Workbooks/Product Analytics using the Click Analytics Plugin/HEART Analytics - All Sections/HEART Analytics - All Sections.workbook

```diff
@@ -848,5 +848,9 @@
       "name": "Technical Validation"
     }
   ],
+  "fallbackResourceIds": [
+    "/subscriptions/96eced6b-b9cc-4575-99ad-ff12cf25085a/resourceGroups/HEART-Scorecard-NonProd/providers/microsoft.insights/components/HeartAppInsightsDemoInstance"
+  ],
+  "fromTemplateId": "community-Workbooks/Product Analytics using the Click Analytics Plugin/HEART Analytics - All Sections",
```

Comment:
the fallback resource ids and template id fields need to be removed (Build validation will catch this too)

Reply:
Done.